import "room_event.proto";

message Event_RoomSay {
	extend RoomEvent {
		optional Event_RoomSay ext = 1002;
	}
	optional string name = 1;
	optional string message = 2;
}
